### PR TITLE
Change default ambient occlusion gamma to 1.8

### PR DESCRIPTION
**Pull request by sfan5**
implements #9865 (compromise) 

The change is subtle but adds more depth to terrain:
screenshot_20200714_135125
<b>⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓⇓</b>
screenshot_20200714_135103

## To do

This PR is Ready for Review.

### CODE REVIEW

**Comment by paramat** [comment]
No objection from me, and looks acceptable in the screenshot. Thanks for compromising.

**Comment by paramat** [comment]
I think we can assume this is approved as the issue had 2 core devs approving a more extreme increase.
I cannot approve as i prefer the current =)

**Review by rubenwardy — APPROVED** [review]

**Review by nerzhul — APPROVED** [review]

**Pull request merged**
